jsapp/js/account/usage.api.ts

```diff
@@ -0,0 +1,26 @@
+import {fetchGet} from 'jsapp/js/api';
+
+interface AssetUsage {
+  asset: string;
+  asset__name: string;
+  submission_count_current_month: number;
+  submission_count_all_time: number;
+  nlp_usage_current_month: any,
```

Comment:
Use unknown instead of any. Then if we end up using this later, TS will error and we can substitute in the right value. With any, it won't error and we don't think to go back and update it.